annotations/views.py

```diff
@@ -268,11 +268,16 @@ def list_user(request):
     sort_dict = {"user_name":"username", "name":"full_name",
      "aff":"affiliation", "loc":"location"}
 
+    srch_term = request.GET.get('srch-term')
     sort = request.GET.get('sort', 'user_name')
 
     sort_column = sort_dict[sort]
 
-    queryset = VogonUser.objects.exclude(id = -1).order_by(sort_column)
+    if(srch_term):
+        queryset = VogonUser.objects.filter(username__contains = srch_term).exclude(id = -1).order_by(sort_column)
+    else:
+        queryset = VogonUser.objects.exclude(id = -1).order_by(sort_column)
```

Comment:
Erick, how should I go about this. Should I create a regex for filtering? Because for contains I cannot use any default value.